nox.py

```diff
@@ -30,13 +30,13 @@ def docs(session):
     # Install Sphinx and also all of the google-cloud-* packages.
     session.chdir(os.path.realpath(os.path.dirname(__file__)))
     session.install('Sphinx >= 1.6.2', 'sphinx_rtd_theme')
+    session.install('.')
```

Comment:
I just verified that `nox -e docs` fails without that line (dammit).  OIC, it is because `docs/conf.py` sniffs the version via `pkg_resources`.